test/optional.js

```diff
@@ -17,9 +17,9 @@ test("aria: base", async () => {
 });
 
 test("data: base", async () => {
-  let el = h`<input :data="{a:1, fooBar:2}"/>`;
-  let params = sprae(el);
-  is(el.outerHTML, `<input data-a="1" data-foo-bar="2">`);
+  let el = h`<input :data="{a:1, fooBar:2, UpperCase:3}"/>`;
+  sprae(el);
+  is(el.outerHTML, `<input data-a="1" data-foo-bar="2" data-upper-case="3">`);
```

Comment:
Do you think that's desired behavior? Is seems default renders as `data--upper-case=3` which is kind of ok, no?

Reply:
It's just foolproof.
But decide for yourself.

Reply:
I mean best foolproof is just relying on standard ways, so we're sort of not responsible.
One of aspects I try to care is reasonably minimal size, if we foolproof cases unnecessarily it will bloat the thing. So let's better keep this out unless we actually have a fool who we need to proof.